src/components/Image/Image.visual.tsx

```diff
@@ -48,6 +48,15 @@ visualize('Image', () => {
       snap('with onError', (done) => (
         <ImageWrapper src={invalidSrc} onDone={done} />
       ));
+      snap('with blurry loading', (done) => (
+        <Image
+          src={src}
+          width={200}
+          height={200}
```

Comment:
Currently we have to pass the dimensions to render a media image, so we cannot take the screenshot without them (even though they are not related to the test)